master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala

```diff
@@ -320,6 +324,13 @@ private[celeborn] class Master(
         "send-application-meta")
     }
 
+    runTransportMessagesStaticBlockerTask = runTransportMessagesThread.submit(
```

Comment:
Maybe we can use a TransportMessagesHelper to handle pre-register. Besides this,is pre-register really needed For LifecycleManager/Master/Worker?

Reply:
@RexXiong, it's really needed for Master, Worker and LifecycleManager, where `TransportMessages` is built.